e2e/chart/snapshot/values.yaml

```diff
@@ -0,0 +1,3 @@
+namePrefix: snapshot
+finalizers:
+- controller.snapshot/finalizer
```

Comment:
Not sure... The snapshot CRD and controller are the parts of the sub-project with own Makefile targets and actual code for them. Probably it will be better to rely on those targets.

Reply:
This chart created for testing purposes, I prefer to use this separate chart.